frontend/src/store/booking/bookingReducer.js

```diff
@@ -0,0 +1,24 @@
+import { actionType } from "./bookingActions";
+
+const initialState = {
+  numberOfSeats: null,
+};
+
+const bookingReducer = (state, action) => {
```

Comment:
Could comments be potentially helpful here

Reply:
Hmm for what?

Reply:
To explain what booking reducer does or maybe I'm just confused

Reply:
Done

Reply:
Please merge when you can @nidhineshnand